### PR TITLE
Replace feedcenter with raven/sentry.

### LINKED ISSUE #118: Replace feedcenter with Sentry.

### PR DESCRIPTION
**Pull request by busykoala**
Close https://github.com/4teamwork/bobtemplates.4teamwork/issues/118

### CODE REVIEW

**Comment by busykoala** [comment]
@mbaechtold, sorry for the early review request. I had to also do changes in lawgiver (https://github.com/4teamwork/ftw.lawgiver/pull/93), remove `unittest2` from this package and update the workflows because of `ftw.file`.
It it ready for review now ;-)

File: setup.py [hunk 1]
```diff
@@ -4,7 +4,6 @@
 
 extras_require = {
     'test': [
-        'unittest2',
     ],
 }
```
Comment (mbaechtold) [inline setup.py]: The `test` extra is quite empty now. We could remove it altogether. You would need to change the `setup()` method too.

**Review by mbaechtold — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (busykoala) [reply setup.py]: 😄 good point!

**Review by mbaechtold — APPROVED** [review]
Thanks a bunch 👍

**Pull request merged**